### PR TITLE
Add hooks displayAdminThemesListAfter and displayModuleConfigureExtraButtons for 8.0.0

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add hooks displayAdminThemesListAfter and displayModuleConfigureExtraButtons for autoupgrade of 8.0.0
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | See https://github.com/PrestaShop/PrestaShop/pull/27469 and https://github.com/PrestaShop/PrestaShop/pull/27461
| How to test?      | -
| Possible impacts? | -

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]

**Review by Progi1984 — DISMISSED** [review]

File: upgrade/sql/8.0.0.sql
Comment (PierreRambaud) [inline upgrade/sql/8.0.0.sql:27]:
```suggestion
  (NULL, 'displayCheckoutSummaryTop', 'Cart summary top', 'This hook allows you to display new elements in top of cart summary', '1'),
  (NULL, 'displayAdminThemesListAfter', 'BO themes list extra content', 'This hook displays content after the themes list in the BackOffice', '1'),
  (NULL, 'displayModuleConfigureExtraButtons', 'Module configuration - After toolbar buttons', 'This hook allows to add toolbar''s additional content on module configuration page', '1')
```

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
The SQL is broken ^^

**Comment by sowbiba** [comment]
> The SQL is broken ^^

oups. Fixed

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by AureRita — CHANGES_REQUESTED** [review]
Hi @sowbiba, 

I tested it and hooks that you add for autoupgrade isn't show, 

as you can see on this video 

Can't change hooks of Autoupgrade 
https://user-images.githubusercontent.com/100699445/158362434-e9205f2c-4538-4744-b098-479639af519a.mp4

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by PululuK — DISMISSED** [review]

**Review by PululuK — DISMISSED** [review]

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Unfortunately, this PR needs to be rebased

**Review by Progi1984 — APPROVED** [review]

**Comment by AureRita** [comment]
hi @sowbiba 

I don't see any label, it is waiting for QA ? =)

**Comment by sowbiba** [comment]
We normally have to wait for the 2nd approval. But, as it's just a rebase, you can start your testing ;)

**Comment by AureRita** [comment]
Hi @sowbiba 

I do an autoupdate from 1.7.8.5 to 8.0.0 and we have some exceptions : 

On BO : 
Modules > Modules Managers : 
image

Modules > Modules Catalog 
image

instead of : 
image
(it's known issue) 

Design > Theme Catalog
image

Design > Positions 
image

Payment > Payment Methods : 
image

Payment > Preferences 
image

International > Translations 
image

Marketing 
image

Because of the exception on Design > Position, I can't check the hook on autoupgrade, I see that's a common issue between all these pages 

Thank you for persisting on this pr

**Comment by sowbiba** [comment]
Hello @AureRita , you choosed a non working module to install (Gamiphy)

If you want to test a working module with configure page, you can use
[prestashopexamplemodule.zip](https://github.com/PrestaShop/autoupgrade/files/8558291/prestashopexamplemodule.zip)
 or Email Templates Manager in the modules catalog page

**Comment by AureRita** [comment]
Hello @sowbiba 

I tested you PR with an autoupgrade from 1.7.8.5 to 8.0.0, I deleted the module gamiphy (it break the design> position and other tab as you say it) but currently, the module autoupgrade didn't have other hook as you can see :

https://user-images.githubusercontent.com/100699445/165247446-095bfc9c-b5ad-4465-96ba-7a86e7b89167.mp4

**Comment by sowbiba** [comment]
I understand now. My hook is not an autoupgrade hook, it's a BO hook. In the list of hooks Design > Positions, just filter with the names (displayAdminThemesListAfter and displayModuleCOnfigureExtraButtons) to have them

**Comment by sowbiba** [comment]
Thanks @AureRita

**Pull request merged**
